bitcoin_client/setup.cfg

```diff
@@ -0,0 +1,30 @@
+[metadata]
+name = ledger_bitcoin
+version = 0.0.1
```

Comment:
Is there a link with the `__version__` variable in `ledger_bitcoin/btchip/__init__.py`? Version don't match.
If expected, what is the purpose of the latter variable?

Reply:
The btchip folder is (extracted from) this legacy library https://github.com/LedgerHQ/btchip-python
So this is just the version of that library. Since it's a subpackage, I suppose it does no harm to keep __version__ there, if only for historical info?

Reply:
Oh sorry I didn't get it was a pick from another library, despite being properly documented in the `bitcoin_client/ledger_bitcoin/btchip/README.md`.

That's a meaningful info all right.